src/CardContent/FileDetails/FileDetails.types.ts

```diff
@@ -9,6 +9,14 @@ export type FileDetailsProps = {
    * The download button text on search.
    */
   downloadButtonTextOnSearch?: string;
+  /**
+   * The download href link.
+   */
+  downloadLinkHref?: string;
+  /**
+   * The download href link target attribute.
+   */
+  downloadLinkTarget?: string;
```

Comment:
Is there a use case for target? I understood that SCENE would open the download url and then redirect you back to the current page. When would we want that to be in a new tab?

Just checking we're not adding unnecessary support here.

Reply:
Perhaps. I figured if we're giving href support and using an anchor then we should give basic attribute support. I don't think they will be using it for the current story.